SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift

```diff
@@ -48,7 +48,7 @@ class AppCenterViewController : NSViewController, NSTextFieldDelegate, NSTextVie
   override func viewDidLoad() {
     super.viewDidLoad()
     installIdLabel?.stringValue = appCenter.installId()
-    appSecretLabel?.stringValue = (UserDefaults.standard.object(forKey: kMSAppSecret) ?? appCenter.appSecret()) as! String
+    appSecretLabel?.stringValue = prodAppSecret()
```

Comment:
This is an app secret which is only available for App Center. You don't need to get PROD app secret based on start up mode. `appCenter.appSecret()` should be enough.